benchmark.md

```diff
@@ -0,0 +1,69 @@
+# Indexer Benchmarks
+
```

Comment:
Add a quick summary at the top, for example.

**TL/TR:** Indexing jsonb data is more efficient than adding new columns to the table schema.

Reply:
Actually, if we are planning to use this doc for all the benchmark lessons, we can do like a table of contents. 
Summarize the lesson in 1 line and add a link to the detailed section.